package.json

```diff
@@ -2,26 +2,21 @@
   "name": "@kodingdotninja/chakra-icons",
   "version": "0.0.0",
   "workspaces": [
-    "packages/*",
-    "website",
-    "!website/**",
-    "!packages/tsconfig/**",
```

Comment:
This is the rules by `multi-semantic-repo` that's mean, the project in our workplace NOT to publish. 

` "!packages/tsconfig/**"` (not released)

but if you want to release, go for it.

Reply:
Updated the globs, hopefully this works.

Reply:
Thank you @grikomsn